zingo-testutils/src/lib.rs

```diff
@@ -681,6 +681,69 @@ pub mod scenarios {
         )
     }
 
+    pub async fn funded_orchard_with_3_txs_mobileclient(
+        value: u64,
+    ) -> (RegtestManager, ChildProcessHandler) {
+        let mut scenario_builder = setup::ScenarioBuilder::build_configure_launch(
+            Some(REGSAP_ADDR_FROM_ABANDONART.to_string()),
+            None,
+            Some(20_000),
+        )
+        .await;
+        let faucet = scenario_builder
+            .client_builder
+            .build_new_faucet(0, false)
+            .await;
+        let recipient = scenario_builder
+            .client_builder
+            .build_newseed_client(HOSPITAL_MUSEUM_SEED.to_string(), 0, false)
+            .await;
+        increase_height_and_sync_client(&scenario_builder.regtest_manager, &faucet, 1)
+            .await
+            .unwrap();
+        // received from a faucet
+        faucet
+            .do_send(vec![(
+                &get_base_address!(recipient, "unified"),
+                value,
+                None,
+            )])
+            .await
+            .unwrap();
+        increase_height_and_sync_client(&scenario_builder.regtest_manager, &recipient, 1)
+            .await
+            .unwrap();
+        // send to a faucet
+        recipient
+            .do_send(vec![(
+                &get_base_address!(faucet, "unified"),
+                value.checked_div(10).unwrap(),
+                None,
+            )])
+            .await
+            .unwrap();
```

Comment:
Why is the following `increase_height_and_sync_client` called?   Test time is important, faster tests are better.

Reply:
one of your comments was a block is needed to be generated to confirm the previous send right? and the recipient needs to sync for next send

Reply:
No.   The recipient can GOSSIP any number of sends that it knows it has funding for.

In this case the two separate receivers can be updated as the 1st and 2nd members of the `vec![` in a single `do_send`..   OR `do_send` can be called twice, and then the sync can happen after.

The critical point to understand is that there's multistep process:

1.  transaction construction (can be completely offlinne)
2.  transaction GOSSIP, zcashd nodes learn about the transaction, but NO BLOCKS HAVE CONFIRMED
3. confirmed transaction, the transaction is *in* a block

The `LightClient.do_send` method does steps 1 and 2, but (of course) it can't do step three which is up to the miners.   Clarifying this mental model is really important for `zingolib` development and use.